examples/allocation/tinygo-malloc/testdata/greet.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"fmt"
+	"unsafe"
+)
+
+// #include <stdlib.h>
+import "C"
+
+// main is required for TinyGo to compile to Wasm.
+func main() {}
+
+// greeting gets a greeting for the name.
+func greeting(name string) string {
+	return fmt.Sprint(name, "!")
```

Comment:
can you please change this to exactly the same as the other thing as having differences makes the benchmarks hard to assess

e.g.

```
// greeting gets a greeting for the name.
func greeting(name string) string {
	return fmt.Sprint("Hello, ", name, "!")
}
```

Reply:
done